views_api.py

```diff
@@ -68,7 +69,7 @@ async def api_links(
 
 @lnurlp_ext.get("/api/v1/links/{link_id}", status_code=HTTPStatus.OK)
 async def api_link_retrieve(
-    r: Request, link_id, wallet: WalletTypeInfo = Depends(get_key_type)
+    r: Request, link_id, key_info: WalletTypeInfo = Depends(require_invoice_key)
```

Comment:
nitpick: add type to `link_id`

Reply:
why not allow `adminkey` for this GET operation?

Reply:
add types